src/SubscribeMe/Subscriber/MailjetSubscriber.php

```diff
@@ -68,10 +82,78 @@ public function subscribe(string $email, array $options, array $userConsents = [
                     return $body['Data'][0]['ContactID'];
                 }
             }
-        } catch (RequestException $exception) {
+        } catch (ClientExceptionInterface $exception) {
             throw new CannotSubscribeException($exception->getMessage(), $exception);
         }
 
         return false;
     }
+
+    /**
+     * @see https://dev.mailjet.com/email/guides/send-api-v31/#use-templating-language
+     * @param array<EmailAddress> $emails
+     * @param array $variables
+     * @param string $templateEmail
+     * @return string
+     */
+    public function sendTransactionalEmail(array $emails, array $variables, string $templateEmail): string
+    {
+        $recipients = array_map(function (EmailAddress $emailAddress) {
```

Comment:
Unnecessary `$recipients` extracted variable.

You can injected it directly in your body.